src/autogenerated/java/net/minestom/server/map/MapColors.java

```diff
@@ -0,0 +1,201 @@
+package net.minestom.server.map;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * AUTOGENERATED by MapColorsGenerator
+ */
+public final class MapColors {
```

Comment:
Not sure I like the name clash with this and `MapColor`. Can't quite think of a better option though..